prime-router/src/main/kotlin/history/db/ReportGraph.kt

```diff
@@ -131,6 +136,33 @@ class ReportGraph(
         }
     }
 
+    data class Item(
```

Comment:
I think kdocs would be useful here and elsewhere for newly added methods. For example, what is an Item supposed to model?

Reply:
Yea... I need to add kdocs across the board before merging